notebook/static/notebook/js/actions.js

```diff
@@ -462,38 +462,23 @@ define(function(require){
                 env.notebook.show_command_palette();
             }
         },
-        'show-all-line-numbers': {
```

Comment:
Good point. I added back the show/hide line numbers as well as adding show/hide actions for header and toolbar.